ts/src/phemex.ts

```diff
@@ -444,6 +444,20 @@ export default class phemex extends Exchange {
                 'transfer': {
                     'fillResponseFromRequest': true,
                 },
+                'triggerPriceTypesMap': {
```

Comment:
@ttodua this concept is not unified yet and I would not do it for now

Reply:
i am not fully sure how it's related whether it's unified or not. it's just an "option" to parse and make code cleaner. even if it changes, what will be the problem.

Reply:
can we keep this PR so maybe I can collect such PRs and then you can see the whole image better

Reply:
second question - if I remove `triggerPriceTypesMap` from the edits, are other changes ok? i think they fix the bug

Reply:
- there were `stopLossPrice` & `takeProfitPrice` in old format parsed down below, which I moved in the top and routed them through unified way
- there was mistake about `type` param name in example (and it wasn't used anywhere in request building process)